### PR TITLE
Build pushes to all branches not only the master branch

### PR DESCRIPTION
**Pull request by relekang**

### CODE REVIEW

File: frigg/builds/models.py [hunk 1]
```diff
@@ -87,16 +87,23 @@ def start_build(self, data):
...
-            build_number=self.last_build_number + 1,
-            pull_request_id=data['pull_request_id'],
             branch=data['branch'],
             sha=data['sha'],
-            author=data['author'],
-            message=data['message']
-        ).start()
+            author=data['author']
+        )
+        build.pull_request_id = data['pull_request_id']
```
Comment (frecar) [inline frigg/builds/models.py]: this will produce a key error on push to master?

Comment (frecar) [inline frigg/builds/models.py]: is this 0 on push to master?

Comment (relekang) [inline frigg/builds/models.py]: Yes, from parse_push_payload

**Comment by frecar** [comment]
:shipit:

**Pull request merged**